cppcache/include/geode/CacheableBuiltins.hpp

```diff
@@ -642,6 +642,16 @@ _GF_CACHEABLE_CONTAINER_TYPE_DEF_(HashSetOfCacheableKey,
  */
 _GF_CACHEABLE_CONTAINER_TYPE_(HashSetOfCacheableKey, CacheableLinkedHashSet);
 
+template <>
+inline CacheableKeyPtr CacheableKey::create(int value) {
```

Comment:
Why only these two specializations here? I would also prefer we say `int32_t` rather than `int` to be safe.

Reply:
These were the only 2 that were needed to get compile/tests working.  Alright, I'll switch to `int32_t`.